consumer.go

```diff
@@ -461,9 +461,47 @@ func (c *Consumer) Fetch(batchSize int) ([]*Msg, error) {
 		c.dlsMsgsMutex.Unlock()
 		return msgs, nil
 	}
+
+	c.conn.prefetchedMsgs.lock.Lock()
+	if prefetchedMsgsForStation, ok := c.conn.prefetchedMsgs.msgs[c.stationName]; ok {
```

Comment:
please make sure the station name in here is already lowered case

Reply:
changed `c.stationName` to strings.ToLower(c.stationName)`